packages/synapse-interface/pages/_app.tsx

```diff
@@ -51,10 +52,12 @@ function App({ Component, pageProps }: AppProps) {
                 <PersistGate loading={null} persistor={persistor}>
                   <SegmentAnalyticsProvider>
                     <UserProvider>
-                      <BackgroundListenerProvider>
-                        <Component {...pageProps} />
-                      </BackgroundListenerProvider>
-                      <CustomToaster />
+                      <MaintenanceProvider>
```

Comment:
We have a `BackgroundListenerProvider` to [handle background jobs](https://github.com/synapsecns/sanguine/blob/master/packages/synapse-interface/contexts/BackgroundListenerProvider.tsx) so we don't have to create a new Context & Provider for each one. Let's move all the logic in `MaintenanceProvider` into its own hook and then call it in `BackgroundListenerProvider`.

Reply:
Sounds good! That makes more sense

Reply:
Shifted logic from `MaintenanceProvider` into `useMaintenanceListener` hook in https://github.com/synapsecns/sanguine/pull/2844/commits/62589b304adefad54e4512534f6b66ea46c6faa4